src/transformers/models/bert/modeling_bert.py

```diff
@@ -228,7 +228,7 @@ def __init__(self, config):
         self.position_embedding_type = getattr(config, "position_embedding_type", "absolute")
         if self.position_embedding_type == "relative_key" or self.position_embedding_type == "relative_key_query":
             self.max_position_embeddings = config.max_position_embeddings
-            self.distance_embedding = nn.Embedding(2 * config.max_position_embeddings - 1, self.attention_head_size)
+            self.distance_embedding = nn.Embedding(2 * config.max_position_embeddings + 1, self.attention_head_size)
```

Comment:
We are sadly not really a lento make this correction as it would break existing pretrained checkpoints as the expected shape for a weight would not correspond to the actual saved weight.

Reply:
Okay, i will revert it